src/plugins/lockToVotePlugin/utils/lockToVoteProposalUtils/lockToVoteProposalUtils.ts

```diff
@@ -69,12 +69,15 @@ class LockToVoteProposalUtils {
         const { supportThreshold } = proposal.settings;
         const { votesByOption } = proposal.metrics;
 
-        const parsedSupport = BigInt(tokenSettingsUtils.ratioToPercentage(supportThreshold));
+        const parsedSupport = BigInt(supportThreshold);
+
         const yesVotes = this.getVoteByType(votesByOption, VoteOption.YES);
         const noVotesCurrent = this.getVoteByType(votesByOption, VoteOption.NO);
+
+        // Keeps mental model more closely aligned with token plugin implementation
         const noVotesComparator = noVotesCurrent;
 
-        return (BigInt(100) - parsedSupport) * yesVotes > parsedSupport * noVotesComparator;
+        return (tokenSettingsUtils.ratioBase - parsedSupport) * yesVotes > parsedSupport * noVotesComparator;
```

Comment:
As we're now only creating a big-int, I'd remove the `parsedSupport` const and simply do the wrapping on the calculation, same for the token proposal utils:
```suggestion
        return (tokenSettingsUtils.ratioBase - BigInt(supportThreshold)) * yesVotes > parsedSupport * noVotesComparator;
```